### PR TITLE
Quadren.Arc.Prerelease version PBv1.1.3

### PR DESCRIPTION
**Pull request by rishavnandi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90524)

### CODE REVIEW

**Comment by rishavnandi** [comment]
@mdanish-kh made a new manifest with required updates, do let me know if anything else needs to be corrected!

File: manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+PackageLocale: en-US
+Publisher: Quadren
+PackageName: Arc
+License: MIT License
```
Comment (mdanish-kh) [inline manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```
I can't find the license https://github.com/quadren/arc/ or on the package website https://quadren.github.io/arc/. Can you provide a link to the package's license if it exists? If no solid information about the license is available, this field should be set to `Proprietary`.

File: manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/quadren/arc/releases/download/PBv1.1.3/Arc_Setup.exe
+  InstallerSha256: BAFFAB1BD31FD6303B6B2BDCB250F40935224D6866FF6DFAED7848087348339D
```
Comment (mdanish-kh) [inline manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.installer.yaml:10]:
```suggestion
  InstallerSha256: BAFFAB1BD31FD6303B6B2BDCB250F40935224D6866FF6DFAED7848087348339D
  ProductCode: '{E07B6521-54F8-498E-A095-9BAEBBA98C44}_is1'
```
Although not mandatory for `.exe` installers, but helps WinGet to better match upgrades.

File: manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+PackageLocale: en-US
+Publisher: Quadren
+PackageName: Arc
+License: MIT License
+ShortDescription: Arc Seamless File Transfer
```
Comment (mdanish-kh) [inline manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Arc is a seamless cross-platform file transfer service by Quadren. Fast, secure and 100% free, with support for Windows, MacOS, Android and iOS.
```
ShortDescription should be descriptive and meaningful

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by rishavnandi** [comment]
@mdanish-kh thanks for the detailed feedback! I will make the required changes

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/90524#discussion_r1038759062 Marking as `Needs-Author-Feedback` until the suggestions are addressed.

**Pull request closed without merging**